adding page for livechat

salomon opened the pull request:
adding page for livechat integration

salomon committed:
adding page for livechat

adding page for livechat integration

eskfung commented on _posts/2015-08-13-livechat.md:
```diff
@@ -0,0 +1,13 @@
+title: LiveChat Integration
+categories: integrations
+author: Ryan Salomon
+tags: [livechat, integrations]
```
:+1: even though we're not doing anything with tags

salomon committed:
adding body text to livechat integration

eskfung commented on _posts/2015-08-13-livechat.md:
```diff
@@ -0,0 +1,38 @@
+LiveChat Pre-chat survey filled in<br>
+LiveChat Post-chat survey displayed<br>
+LiveChat Post-chat survey filled in<br>
+LiveChat Ticket form displayed ("ticket form" is displayed when no chat agents are available to chat)
```
Seems like this needs a trailing `<br>`

eskfung commented on _posts/2015-08-13-livechat.md:
```diff
@@ -0,0 +1,38 @@
+
+For simplicity, we’ve given you a property named ‘Performed Chat’. If a user has a value of ‘true’ for this property, you know they’ve performed a chat.  You can then use this property across all of your Kissmetrics reports to see how LiveChat users differ from your other users across all the events you’re already tracking.  
+
+![LiveChat 1][livechat_image_1]<br>
```
The trailing `<br>` in this case is probably unnecessary because you've already put this in its own paragraph (`<p>`) by having a line break before and after.

eskfung commented on _posts/2015-08-13-livechat.md:
```diff
@@ -0,0 +1,38 @@
+
+On top of this functionality, we’ve also included the ability to segment by which agent was chatting with the user.  The property for this is named ‘agent’.  If ‘Ryan’ was the person in your organization they were speaking with, then we’d tag them with a property of ‘agent: ryan’.
+
+![LiveChat 2][livechat_image_1]<br>
```
Same comment as above.

eskfung commented on _posts/2015-08-13-livechat.md:
```diff
@@ -0,0 +1,38 @@
+
+![LiveChat 2][livechat_image_1]<br>
+
+We’re super excited to bring this to you in a partnership with LiveChat! Click [here][1] to head over their page for directions on how to set up.  Don't worry, we’ve made set-up nice and easy!
```
over _to_ their

salomon committed:
bullet list for livechat list of events

eskfung committed:
Residual style changes

eskfung commented:
@salomon :+1: :shipit: when you're ready

salomon committed:
copy change on 'agent' and image switch

copy image changed out because it needed to reflect change in the
property value that is stored for ‘agent’ property.

salomon committed:
Merge remote-tracking branch 'origin/livechat_integration' into livechat_integration

Conflicts:
	_posts/2015-08-13-livechat.md
